ts/nni_manager/training_service/reusable/environments/dlcEnvironmentService.ts

```diff
@@ -76,8 +80,12 @@ export class DlcEnvironmentService extends EnvironmentService {
                     environment.setStatus('SUCCEEDED');
                     break;
                 case 'FAILED':
+                    // the job create failed,we will sleep(60) to create new job
+                    await this.sleep(60);
+                    this.log.debug(`await 60s to create new job,DLC: job ${environment.id} is failed!`);
                     environment.setStatus('FAILED');
-                    return Promise.reject(`DLC: job ${environment.envId} is failed!`);
+                    deferred.reject(`DLC: job ${environment.id} is failed!`);
```

Comment:
The reject parameter should be an `Error`.

Reply:
OK,I can delete it
I just see the openpaienvionmentservice ,so i use it